epub33/core/index.html

```diff
@@ -1368,20 +1368,29 @@ <h5>HTML <code>img</code> fallbacks</h5>
 									href="#sec-manifest-fallbacks">manifest fallback</a>.</li>
 						</ul>
 					</section>
+
 					<section>
 						<h5>HTML <code>script</code> element</h5>
 
-						<p>Although the [[html]] [^script^] element lacks intrinsic fallback capabilities, it is not
-							required that [=EPUB creators=] provide fallbacks for <a data-cite="html#data-block">data
-								blocks</a> [[html]]. As the <code>script</code> element does not represent user content,
-							data blocks are not rendered unless manipulated by script.</p>
+						<p>Although data blocks have a separate MIME media type [[rfc2046]] from their containing
+							[=XHTML content document=], it is not possible to provide intrinsic fallbacks as no such mechanisms are specified for the [[html]] [^script^] element. It is also not possible to provide
+							manifest fallbacks because data blocks are always embedded in a <code>script</code>
+							element.</p>
```

Comment:
s/because data blocks are always embedded in a `script` element/because data blocks cannot be defined as standalone files in the EPUB container but are always embedded in XHTML contents document as inline `script` elements./

Reply:
Works for me. I stripped the "in XHTML content documents" part to shorten it a bit because this is implied by the section and nature of the element.